app/mailers/katello/subscription_mailer.rb

```diff
@@ -4,12 +4,14 @@ class SubscriptionMailer < ApplicationMailer
 
     def subscription_expiry(options)
       user = ::User.find(options[:user])
+      days_from_now = options[:query]
+
       ::User.as(user.login) do
-        @pools = Katello::Pool.readable.expiring_in_days(30)
+        @pools = Katello::Pool.readable.expiring_in_days(days_from_now)
       end
 
       set_locale_for(user) do
-        mail(:to => user.mail, :subject => _("Subscriptions expiring soon"))
+        mail(:to => user.mail, :subject => _("You have subscriptions expiring within #{days_from_now} days"))
```

Comment:
For translations we shouldn't use interpolation, but this format instead (that's the reason for the test failure):

```
_("You have subscriptions expiring within %s days") % days_from_now
```

Reply:
thanks!  I was about to ask about that :smile: